ansible/roles/ocp4-workload-openbanking-sandbox/tasks/3amp-deploy-full-api.yml

```diff
@@ -13,13 +13,18 @@
     url: "{{ obsandbox_api_crds_base_url + '/backend-' + _api_resource_id + '.yml' }}"
     dest: "{{ _crd_template.path }}"
 
+# Using 'cat' because look('file') or lookup('template') fails on remote host
+- name: Read Backend CRD tempfile '{{ _crd_template.path }}' into '_crd_content' variable
+  shell: cat "{{ _crd_template.path }}"
```

Comment:
It would be better to use the **slurp** module here.

Reply:
Ok, did the changes.